index.org

```diff
@@ -3,14 +3,24 @@
 #+AUTHOR: Manoel Vilela, Lucas Vieira
 #+DATE: <2018-02-10 Sat>
 #+TITLE: Common Lisp Brasil
+#+LANGUAGE: bt-br
```

Comment:
pt-br

Reply:
é incrível dizer mas... o language-alist do babel realmente fez essa brutalidade de colocar essa key de associação pra brazilian :[ 
![image](https://user-images.githubusercontent.com/7642878/36103028-0ca8d20a-1006-11e8-8f6c-d267b5abf7a3.png)

Reply:
se bem que isso é do latex... :thinking:

Reply:
No fim não fez diferença da linguagem para HTML. To pensando aqui no que fazer. Eu queria era traduzir o TOC de Table of Contents para Sumário na geração do html